Extensions/ExampleJsExtension/JsExtension.js

```diff
@@ -33,6 +33,27 @@ module.exports = {
       new gd.PropertyDescriptor(/* defaultValue= */ "1")
         .setLabel(_('Opacity of the effect (between 0 and 1)'))
         .setType('number')
+        .setDescription(_("This is an optional description."))
+    );
+    dumyEffectProperties.set(
+      'someImage',
+      new gd.PropertyDescriptor(/* defaultValue= */ "")
+        .setLabel(_('Some image for the effect'))
+        .setType('resource')
+        .addExtraInfo('image')
+        .setLabel(
+          _("Image resource (won't be used, just for demonstration purpose)")
+        )
+    );
+    dumyEffectProperties.set(
+      'someColor',
+      new gd.PropertyDescriptor(/* defaultValue= */ "#0022FF")
+        .setLabel(_('Some color for the effect'))
+        .setType('color')
+        .setLabel(
```

Comment:
There are two setLabel()

Reply:
Good catch ;) I've fixed this in master.